databricks/koalas/frame.py

```diff
@@ -2145,6 +2145,159 @@ def sort_values(self, by, ascending=True, inplace=False, na_position='last'):
         else:
             return kdf
 
+    # TODO: Not Supported:  keep = First
+    def nlargest(self, n, columns, keep='first'):
+        """
+        Return the first `n` rows ordered by `columns` in descending order.
+        Return the first `n` rows with the largest values in `columns`, in
+        descending order. The columns that are not specified are returned as
+        well, but not used for ordering.
+        This method is equivalent to
+        ``df.sort_values(columns, ascending=False).head(n)``, but more
+        performant.
+
+        Parameters
+        ----------
+        n : int
+            Number of rows to return.
+        columns : label or list of labels
+            Column label(s) to order by.
+        keep : Not yet supported
```

Comment:
this should be

```
keep :  {‘first’, ‘last’, ‘all’}, default ‘first’
    Only the default value first is supported
```

Reply:
Shouldnt it be 'all' that is supported and not first?